src/cli/commands/autoclean.js

```diff
@@ -67,7 +68,7 @@ export async function clean(
   removedFiles: number,
   removedSize: number,
 }> {
-  const loc = path.join(config.cwd, CLEAN_FILENAME);
+  const loc = path.join(config.lockfileFolder, CLEAN_FILENAME);
```

Comment:
nit: seems like you're fixing another issue here and not adding tests for it ;)

Reply:
Test would be a pain, the command would have to run in a subfolder of the fixutre. The `run` helper doesn't work well with this use case.